packages/service/README.md

```diff
@@ -191,7 +192,6 @@ interface BtcApis {
 interface BtcApiBlockchainInfo {
   chain: string;
   blocks: number;
-  headers: number;
   bestblockhash: number;
```

Comment:
Why is the data type of the `bestblockhash` a number? Shouldn't it be a hex string?

Reply:
Fixed at 5ee06ec